src/components/IconList/IconList.tsx

```diff
@@ -27,30 +29,33 @@ export interface IconListItemProps {
   className?: string
   iconSize?: number
   featureStatusType?: FeatureStatusType
+  children?: ReactNode
 }
 
-export const IconListItem: FunctionComponent<IconListItemProps> = ({
+export function IconListItem({
   iconUrl,
   children,
   className,
   iconSize = 40,
   id,
   featureStatusType,
-}) => (
-  <StyledListItem data-testid={id} className={className}>
-    {iconUrl && (
-      <StyledImg alt="" height={iconSize} src={iconUrl} width={iconSize} />
-    )}
-    {featureStatusType && (
-      <StatusIcon
-        alt=""
-        height={20}
-        src={featureStatusType.icon.iconUrl}
-        width={20}
-      />
-    )}
-    {children}
-  </StyledListItem>
-)
+}: IconListItemProps): JSX.Element {
```

Comment:
You don't have to add the `JSX.Element` here, that is the implicit return type already. Also can leave the implicit return.